server.js

```diff
@@ -32,49 +32,46 @@ app.use(helmet());
 app.use(nonceMiddleware, helmet.contentSecurityPolicy({
   directives: {
     defaultSrc: ["'none'"],
-    baseUri: ['https://docs.helpscout.net'],
+    baseUri: ["'none'"],
     scriptSrc: [
       "'self'",
       "'strict-dynamic'",
       (req, res) => {
         return `'nonce-${res.locals.nonce}'`;
       },
-      'https://beacon-v2.helpscout.net',
-      'https://d12wqas9hcki3z.cloudfront.net',
-      'https://d33v4339jhl8k0.cloudfront.net',
+      'https://static.zdassets.com',
+      'https://ekr.zdassets.com',
+      'https://tidepool.zendesk.com',
+      'wss://tidepool.zendesk.com',
+      'wss://*.zopim.com',
     ],
     styleSrc: [
       "'self'",
       'blob:',
-      "'unsafe-inline'",
-      'https://fonts.googleapis.com',
-      'https://beacon-v2.helpscout.net',
-      'https://djtflbt20bdde.cloudfront.net',
+      'unsafe-inline:',
     ],
     imgSrc: [
       "'self'",
       'data:',
-      'https://d33v4339jhl8k0.cloudfront.net',
-      'https://*.gravatar.com',
+      'https://v2assets.zopim.io',
+      'https://static.zdassets.com',
     ],
-    fontSrc: ["'self'", 'data:', 'https://fonts.gstatic.com'],
+    fontSrc: ["'self'", 'data:',],
     reportUri: '/event/csp-report/violation',
-    objectSrc: ['blob:', 'https://beacon-v2.helpscout.net'],
-    workerSrc: ["'self'", 'blob:'],
-    childSrc: ["'self'", 'blob:', 'https://docs.google.com'],
-    frameSrc: ['https://beacon-v2.helpscout.net', 'https://docs.google.com'],
```

Comment:
I missed this earlier:  we actually need to keep the `'https://docs.google.com'` entry in both `frameSrc` and `childSrc`